### PR TITLE
Prometheus Support, Part 4

### PR DESCRIPTION
**Pull request by PhanLe1010**
Added the following metrics:
1. longhorn_node_cpu_capacity_millicpu
2. longhorn_node_cpu_usage_millicpu
3. longhorn_node_memory_capacity_bytes
4. longhorn_node_memory_usage_bytes

We use the above metrics to detect when there is cpu/memory high pressure on worker nodes. This will help when debugging cases such as engines die unexpectedly due to cpu pressure. 

Also, change the metric name `longhorn_volume_usage_bytes` to `longhorn_volume_actual_size_bytes` to avoid the confusion between the size of the filesystem (usage_bytes) vs used space by Longhorn block device (actual size)

longhorn/longhorn#1180

### CODE REVIEW

File: monitoring/volume_collector.go [hunk 1]
```diff
@@ -41,8 +41,8 @@ func NewVolumeCollector(
 
 	vc.sizeMetric = metricInfo{
 		Desc: prometheus.NewDesc(
-			prometheus.BuildFQName(longhornName, subsystemVolume, "usage_bytes"),
-			"Used storage space in bytes for this volume",
+			prometheus.BuildFQName(longhornName, subsystemVolume, "actual_size_bytes"),
```
Comment (yasker) [inline monitoring/volume_collector.go:44]: Separate this change into another commit.

File: monitoring/node_collector.go [hunk 2]
```diff
@@ -49,33 +59,90 @@ func NewNodeCollector(
...
-		nc.logger.WithError(err).Warn("error during scrape")
-		return
-	}
+	var wg sync.WaitGroup
+
+	wg.Add(1)
+	go nc.collectNodeStatus(ch, &wg)
+
+	wg.Add(1)
+	nc.collectTotalNumberOfNodes(ch, &wg)
```
Comment (yasker) [inline monitoring/node_collector.go:121]: hmm? Those are single-threaded so why you need waitgroup? Did you mean to call those as goroutines?

Also, you can avoid passing the waitgroup into the function, since waitgroup is just a flow control e.g.
``` 
wg.Add(1)
go func() {
    defer wg.Done()
    call_func()
}()
```

**Review by yasker — CHANGES_REQUESTED** [review]
Also, change the commit title to something else. Otherwise, it's hard to tell from the commit title what does this commit do. (it was the same as previous commits)

File: monitoring/node_collector.go (hunk 2 above)
Comment (PhanLe1010) [reply monitoring/node_collector.go]: My bad. Yes I meant to call those funcs as goroutines. Also fixed to avoid passing the waitgroup.

File: monitoring/volume_collector.go (hunk 1 above)
Comment (PhanLe1010) [reply monitoring/volume_collector.go:44]: Thank you. Done

**Comment by PhanLe1010** [comment]
Changed the commit name

**Review by yasker — APPROVED** [review]
